app/controllers/multifactor_auths_controller.rb

```diff
@@ -30,7 +32,9 @@ def update
     else
       flash[:error] = t("multifactor_auths.incorrect_otp")
     end
-    redirect_to edit_settings_url
+    continue_path = session.fetch("mfa_redirect_uri", edit_settings_path)
```

Comment:
`continue_path` or `@continue_path`? You use both in this file.

Reply:
My impression was that I had to use @ in create so that the view would be aware of it outside the scope of the function, whereas in update, it redirects in the controller method directly, so I don't need the @?